### PR TITLE
Fix quote on cases

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Smaller line-height
- [x] Smaller position below quote

And remove unused class

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
screen shot 2018-01-31 at 12 09 00

**Comment by RonaldDijkstra** [comment]
Hound, pls wake up

File: source/stylesheets/_cases.scss [hunk 1]
```diff
@@ -177,8 +177,9 @@ section.cases-overview {
   }
 
   .sidebar {
-    background: $white;
     text-align: center;
+    background: $white;
```
Comment (fatboypunk) [inline source/stylesheets/_cases.scss]: why?

Comment (RonaldDijkstra) [reply source/stylesheets/_cases.scss]: A deliberate mistake to see if the Hound would wake up.

Comment (RonaldDijkstra) [reply source/stylesheets/_cases.scss]: It was hanging forever on 'waiting for status to be reported'

**Pull request merged**